lib/node_modules/@stdlib/math/base/special/cinv/src/main.c

```diff
@@ -69,8 +69,7 @@ stdlib_complex128_t stdlib_base_cinv( const stdlib_complex128_t z ) {
 
 	stdlib_reim( z, &re, &im );
 
-	ab = stdlib_base_abs( re ) > stdlib_base_abs( im ) ?
-		 stdlib_base_abs( re ) : stdlib_base_abs( im );
+	ab = fmax( stdlib_base_abs( re ), stdlib_base_abs( im ) );
```

Comment:
@steff456 Need to add back, I believe, `#include <math.h>`. Would you also mind adding a TODO here to replace with `stdlib/math/base/special/max` once implemented?

Reply:
Sure!